### ISSUE TITLE
inspect tool is not working?

### ISSUE THREAD

**Issue by Shemesh** [daem-on/wtherion#20]
in the beta version - inspect tool is not working?

**Comment by daem-on** [comment]
Hi! It's working for me, could you provide more details, for example following this issue template: https://github.com/stevemao/github-issue-templates/blob/master/simple/ISSUE_TEMPLATE.md

**Comment by CaverBruce** [comment]
Shemesh, Inspect activation has changed if you are using the M key to activate. Inspect is now active only while M key is pressed.  See conversation in #18 
Otherwise the eyedropper is working for me - Windows 10 and Brave

**Comment by Shemesh** [comment]
- loading .th2 into https://beta.wtherion.daemonw.com/
- make sure tool "item select" (black arrow) is working
- select the inspect tool (eyedropper)

result:
it does nothing

**Comment by daem-on** [comment]
I'm asking this once again, could you provide some information in your issue like a normal bug report? If you don't know what that looks like, you should use this issue template:
https://github.com/stevemao/github-issue-templates/blob/master/simple/ISSUE_TEMPLATE.md

**Comment by Shemesh** [comment]
sorry, i thought that was clear enough.
here:
Expected Behavior: use inspect tool to choose an item to see its properties
Actual Behavior: using the inspect tool i'm unable to select nothing, inspect tool does nothing.
Steps to Reproduce the Problem:
- loading .th2 into https://beta.wtherion.daemonw.com/
- make sure tool "item select" (black arrow) is working
- select the inspect tool (eyedropper)

Platform: https://beta.wtherion.daemonw.com/

**Comment by daem-on** [comment]
Yes I think you're expecting to be able to select something, when with Inspect you just hover over things and it shows details in the status bar - the bottom right corner.

**Comment by Shemesh** [comment]
oh, ok.
well... this is a weird ux and unexpected behavior, plus it does not bring any real added value.
my suggestion:
- the data in the status bar to be **always** active, no matter which tool i use
- the inspect tool will select an item and show its properties.

**Comment by Shemesh** [comment]
in addition to my suggestions above i suggest the inspect info will also show in which scrap is the item.
